src/sentry/static/sentry/app/views/groupEvents.jsx

```diff
@@ -113,7 +118,11 @@ const GroupEvents = createReactClass({
       loading: true,
       error: false,
     });
-    const query = {...this.props.location.query, limit: 50, query: this.state.query};
```

Comment:
Are we depending on this behavior of spreading `this.props.location.query`? Is there pagination for this view? If so would it break it?

Reply:
Thanks, good point